test/bin/find.js

```diff
@@ -40,6 +40,12 @@ describe('bin', function() {
     delete require.cache[require.resolve('yargs')]
   })
 
+  it('no option', function() {
```

Comment:
Any idea why this does not get mentioned in the PR's [corresponding travis build logs](https://travis-ci.org/sarbbottam/eslint-find-rules/jobs/127115930#L280)?